DistroLauncher/Application.h

```diff
@@ -84,6 +84,9 @@ namespace Oobe
             if (FAILED(hr) && hr != E_NOTIMPL && hr != E_INVALIDARG) {
                 wprintf(L"Installer did not complete successfully.\n");
                 Helpers::PrintErrorMessage(hr);
+                if (appConfig().mustSkipFallback) {
+                    exit(123);
```

Comment:
Why `123`?

I'd rather have some variable at the top of the file, something like `constexpr int EXIT_OOBE_NO_FALLBACK = 123`.

Ideally we'd have a file defining all possible exit values, but since it's only this one I think that'd be overkill.

Reply:
Agreed.